### PR TITLE
Release Hy 1.0a4

### PR DESCRIPTION
**Pull request by Kodiologist**
I've decided that the right order of operations is to release Hy first and Hyrule second, because Hyrule depends on Hy and not the reverse. But the goal is to be practically simultaneous, so I wouldn't merge this before Hyrule was ready, too.

### CODE REVIEW

**Comment by Kodiologist** [comment]
@hylang/core Remember, I need two approvals on this (and on hylang/hyrule#19) to merge.

**Review by allison-casey — APPROVED** [review]

**Review by brandonwillard — APPROVED** [review]

**Comment by Kodiologist** [comment]
Thanks, guys. I'll make this happen once I have enough approvals for Hyrule, too.

**Pull request merged**
